build/ToolsetPackages/project.lock.json

```diff
@@ -0,0 +1,1171 @@
+      "sha512": "7ANNTRnLXBAwYX8bQdDD23W3Fxotur5HyW/HVhColCEOM6eg5BLlbqjtXKi+WOqCxGeZY44ymqnj9t49Q7eZMw==",
+      "type": "Package",
+      "files": [
+        "_rels/.rels",
+        "FakeSign.nuspec",
+        "tools/FakeSign.exe",
+        "tools/System.Collections.Immutable.dll",
+        "tools/System.Reflection.Metadata.dll",
+        "package/services/metadata/core-properties/0fedffd6c29f4826ac3520f2efaac4c3.psmdcp",
+        "[Content_Types].xml"
+      ]
+    },
+    "Microsoft.Build.Mono.Debug/14.1.0-prerelease": {
+      "sha512": "cRwhmTsVH4Xj5C5ZSmBaP1IyOP7twK02QuDzgk5sw+JCx0dy2yGTeSEd+GE/+hLfDVXu4641hnTgILpT3Qs2tg==",
+      "type": "Package",
+      "files": [
+        "_rels/.rels",
+        "Microsoft.Build.Mono.Debug.nuspec",
+        "lib/Microsoft.Build.dll",
+        "lib/Microsoft.Build.Framework.dll",
+        "lib/Microsoft.Build.Tasks.Core.dll",
+        "lib/Microsoft.Build.Utilities.Core.dll",
+        "lib/MSBuild.exe",
+        "lib/MSBuild.exe.config",
+        "lib/Microsoft.Common.CurrentVersion.targets",
+        "lib/Microsoft.Common.overridetasks",
+        "lib/Microsoft.Common.props",
+        "lib/Microsoft.Common.targets",
+        "lib/Microsoft.Common.tasks",
+        "lib/Microsoft.CSharp.CurrentVersion.targets",
+        "lib/Microsoft.CSharp.targets",
+        "lib/Microsoft.Data.Entity.Shim.targets",
+        "lib/Microsoft.NETFramework.CurrentVersion.props",
+        "lib/Microsoft.NETFramework.CurrentVersion.targets",
+        "lib/Microsoft.NETFramework.props",
+        "lib/Microsoft.NETFramework.targets",
+        "lib/Microsoft.ServiceModel.Shim.targets",
+        "lib/Microsoft.VisualBasic.CurrentVersion.targets",
+        "lib/Microsoft.VisualBasic.targets",
+        "lib/Microsoft.VisualStudioVersion.v11.Common.props",
+        "lib/Microsoft.VisualStudioVersion.v12.Common.props",
+        "lib/Microsoft.VisualStudioVersion.v14.Common.props",
+        "lib/Microsoft.WinFx.Shim.targets",
+        "lib/Microsoft.WorkflowBuildExtensions.Shim.targets",
+        "lib/Microsoft.Xaml.Shim.targets",
+        "lib/Workflow.Shim.Targets",
+        "lib/Workflow.VisualBasic.Shim.Targets",
+        "lib/tools/Microsoft.CodeAnalysis.CSharp.Desktop.dll",
+        "lib/tools/Microsoft.CodeAnalysis.CSharp.dll",
+        "lib/tools/Microsoft.CodeAnalysis.Desktop.dll",
+        "lib/tools/Microsoft.CodeAnalysis.VisualBasic.Desktop.dll",
+        "lib/tools/Microsoft.CodeAnalysis.VisualBasic.dll",
+        "lib/tools/Microsoft.CodeAnalysis.dll",
+        "lib/tools/Microsoft.Net.ToolsetCompilers.props",
+        "lib/tools/System.Collections.Immutable.dll",
+        "lib/tools/System.Reflection.Metadata.dll",
+        "lib/tools/VBCSCompiler.exe",
+        "lib/tools/VBCSCompiler.exe.config",
+        "lib/tools/csc.exe",
+        "lib/tools/csc2.exe",
+        "lib/tools/vbc.exe",
+        "lib/tools/vbc2.exe",
+        "src/Framework/AssemblyInfo.cs",
+        "src/Framework/BuildEngineResult.cs",
```

Comment:
You mean the compiler assemblies? Yeah that's not great -- we should try and get them to integrate the Microsoft.Net.Compilers package for this.